FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.h

```diff
@@ -50,6 +51,9 @@ NS_EXTENSION_UNAVAILABLE("Firebase Performance is not supported for extensions."
 /** Current running state of the application. */
 @property(nonatomic, readonly) FPRApplicationState applicationState;
 
+/** Firebase Remote Configuration object */
+@property(nonatomic) FPRConfigurations *configurations;
```

Comment:
Should this be in the header? I don't think anyone else needs to use the app activity tracker's dependency on FPRConfigurations